python/cugraph/cugraph/tests/conftest.py

```diff
@@ -32,9 +36,16 @@ def gpubenchmark():
 
 @pytest.fixture(scope="module")
 def dask_client():
-    client, cluster = start_dask_client(protocol="tcp")
+    n_devices = os.getenv("DASK_NUM_WORKERS", 4)
+    n_devices = int(n_devices)
 
-    yield client
+    visible_devices = ",".join([str(i) for i in range(1, n_devices + 1)])
+
+    cluster = LocalCUDACluster(
+        protocol="ucx", rmm_pool_size="25GB", CUDA_VISIBLE_DEVICES=visible_devices
+    )
+    client = Client(cluster)
+    Comms.initialize(p2p=True)
+    rmm.reinitialize(pool_allocator=True)
 
-    stop_dask_client(client)
-    print("\ndask_client fixture: client.close() called")
+    yield client
```

Comment:
Don't we need the equivalent of `stop_dask_client()` after this line too to properly stop the cluster?

Reply:
fixed